web/modules/custom/joinup_core/src/JoinupRelationManager.php

```diff
@@ -123,4 +134,122 @@ public function getParentElibrary(EntityInterface $entity) {
     return $e_library;
   }
 
+  /**
+   * Retrieves all the users that have the administrator role in a group.
+   *
+   * @param \Drupal\Core\Entity\EntityInterface $entity
+   *   The group entity.
+   * @param array $state
+   *   (optional) An array of membership states to retrieve. Defaults to active.
+   *
+   * @return array
+   *   An array of users that are administrators of the entity group.
+   */
+  public function getGroupOwners(EntityInterface $entity, array $state = [OgMembershipInterface::STATE_ACTIVE]) {
+    $role_id = $entity->getEntityTypeId() . '-' . $entity->bundle() . '-administrator';
+
+    $users = [];
+    foreach ($this->getGroupMemberships($entity, $state) as $membership) {
+      $user = $membership->getUser();
+      if (!empty($user) && $membership->hasRole($role_id)) {
+        $users[$user->id()] = $user;
+      }
+    }
+
+    return array_filter($users);
+  }
+
+  /**
+   * Retrieves all the members with any role in a certain group.
+   *
+   * @param \Drupal\Core\Entity\EntityInterface $entity
+   *   The group entity.
+   * @param array $state
+   *   (optional) An array of membership states to retrieve. Defaults to active.
+   *
+   * @return array
+   *   An array of users that are members of the entity group.
+   */
+  public function getGroupUsers(EntityInterface $entity, array $state = [OgMembershipInterface::STATE_ACTIVE]) {
+    $users = array_map(function (OgMembershipInterface $membership) {
+      $user = $membership->getUser();
+      return empty($user) ? NULL : $user->id();
+    }, $this->getGroupMemberships($entity, $state));
+
+    return array_filter($users);
+  }
+
+  /**
+   * Retrieves all the memberships of a certain entity group.
+   *
+   * @param \Drupal\Core\Entity\EntityInterface $entity
+   *   The group entity.
+   * @param array $state
+   *   (optional) An array of membership states to retrieve. Defaults to active.
+   *
+   * @return \Drupal\og\OgMembershipInterface[]
+   *   The memberships of the group.
+   */
+  public function getGroupMemberships(EntityInterface $entity, array $state = [OgMembershipInterface::STATE_ACTIVE]) {
+    /** @var \Drupal\og\OgMembershipInterface[] $memberships */
+    $memberships = $this->entityTypeManager->getStorage('og_membership')->loadByProperties([
+      'state' => $state,
+      'entity_type' => $entity->getEntityTypeId(),
+      'entity_id' => $entity->id(),
+    ]);
+
+    return $memberships;
+  }
```

Comment:
Nope :) it does not! and we stumbled across a few times in the past. Quite weird, no? 
It's good that Francesco put it in a helper class.